src/Api.ts

```diff
@@ -0,0 +1,53 @@
+import Predicates from './Predicates';
+import { RequestHandler, RequestCallback } from './request';
+import { Document } from './documents';
+import { ApiCache } from './cache';
+import ResolvedApi, { ApiData } from './ResolvedApi';
+import HttpClient from './HttpClient';
+
+export interface ApiOptions {
+  accessToken?: string;
+  requestHandler?: RequestHandler;
+  req?: any;
+  apiCache?: ApiCache;
+  apiDataTTL?: number;
+  proxyAgent?: any;
+}
+
+export default class Api {
+  url: string;
+  options: ApiOptions;
+  apiDataTTL: number;
```

Comment:
deja present dans les options d'API

Reply:
this one is `this.options.apiDataTTL || 5;` (line 30)

Reply:
ah yes alright, I get it.